### PR TITLE
add Ubuntu 20.10 Groovy Gorilla

### PR DESCRIPTION
**Pull request by vicamo**

### CODE REVIEW

**Comment by vicamo** [comment]
Note https://partner-images.canonical.com/core/groovy is not yet available.

**Comment by tianon** [comment]
Nice, thanks! :+1:

File: rolling [hunk 1]
```diff
@@ -1 +1 @@
-focal
+groovy
```
Comment (tianon) [inline rolling:1]: This shouldn't be updated until `groovy` is considered released, right?

cc @mwhudson @philroche

**Comment by mwhudson** [comment]
Right, my understanding is: latest -> lts, rolling -> most recent release,
devel- > development series

File: rolling (hunk 1 above)
Comment (philroche) [reply rolling:1]: I agree

Comment (vicamo) [reply rolling]: Right. Thank you for catching this.

**Comment by tianon** [comment]
Looks good, thanks!

**Pull request merged**